lib/u3d/installer.rb

```diff
@@ -272,6 +277,35 @@ def install_xz(file, installation_path: nil)
       UI.success 'Installation successful'
     end
 
+    def install_pkg(file, installation_path: nil)
+      raise 'Missing installation_path' unless installation_path
+      raise 'Only able to install pkg on top of existing Unity installs' unless File.exist? installation_path
+      raise 'Missing 7z' if `which 7z`.empty?
+
+      Dir.mktmpdir do |tmp_dir|
+        UI.verbose "Working in tmp dir #{tmp_dir}"
+
+        command = "7z -aos -o#{tmp_dir.shellescape} e #{file.shellescape}"
+        U3dCore::CommandExecutor.execute(command: command)
+
+        path = pkg_install_relative_path("#{tmp_dir}/PackageInfo")
```

Comment:
Would it be cleaner to wrap this call and the next one into a single method? ie

```ruby
path = pkg_install_path(installationPath, "#{tmp_dir}/PackageInfo")
```

Reply:
One is completely independent of having an installationPath in my head.

Reply:
Changed my mind. Fixing.